.NET wrapper for SDK (#193)
bitwarden/sdk: languages/csharp/Bitwarden.Sdk/README.md

Reviewer: I think we should move this up to the parent directory.

Author: readme file is part of Bitwarden.Sdk project, I can move the file but then when you open the solution (I'm using Rider) you can see the file

Reviewer: I know, the purpose of moving it to `languages/csharp` is to make it more visible to people browsing the repository.

Author: ok, done

> # Bitwarden .NET SDK

Reviewer: We should be explicit that this is only the secrets manager SDK.
```suggestion
# Bitwarden Secrets Manager SDK
```

Author: done

> The Bitwarden SDK (Software Development Kit) is a set of programming tools, libraries, and resources provided by Bitwarden for developers to integrate Bitwarden's password management and security capabilities into their own applications, services, or platforms. The SDK facilitates the seamless integration of Bitwarden's functionality into third-party software, enabling users to securely manage their passwords, credentials, and sensitive information across various digital environments.

Reviewer: I think we can simplify this.
```suggestion
.NET bindings for interacting with the
[Bitwarden Secrets Manager](https://bitwarden.com/products/secrets-manager/). This is a beta release
and might be missing some functionality.
```

Author: done

## Create access token

> - to create access token go to [access-tokens help page]

Reviewer: ```suggestion

Review the help documentation on [Access Tokens](https://bitwarden.com/help/access-tokens/).
```

Author: done

## Usage code snippets

### Create new Bitwarden client

> ```

Reviewer: Prettier wants a newline before code blocks, and we should mark them as `csharp`.
```suggestion

```csharp
```

Author: done

const string accessToken = "<access-token>";
using var bitwardenClient = new BitwardenClient();
var loginResponse = bitwardenClient.AccessTokenLogin(accessToken);
```

### Create new project

> - Organization is created in the UI
> ```

Reviewer: ```suggestion

```csharp
```

const string organizationIdStr = "<organization-id>";
var organizationId = Guid.Parse(organizationIdStr);
var responseForProjectResponse = bitwardenClient.Projects().Create(organizationId, "TestProject");
```

### List all projects

> ```

Reviewer: ```suggestion

```csharp
```

var response = bitwardenClient.Projects().List(organizationId);
```

### Update project

> ```

Reviewer: ```suggestion

```csharp
```

var projectId = responseForProjectResponse.Data.Id;
responseForProjectResponse = bitwardenClient.Projects().Get(projectId);
responseForProjectResponse = bitwardenClient.Projects().Update(projectId, organizationId, "TestProjectUpdated");
```

### Add new secret

> ```

Reviewer: ```suggestion

```csharp
```

var key = "key";
var value = "value";
var note = "note";
var responseForSecretResponse = bitwardenClient.Secrets().Create(key, value, note, organizationId, new Guid[]{projectId});
var secretId = responseForSecretResponse.Data.Id;
```

### List secrets

> ```

Reviewer: ```suggestion

```csharp
```

var responseForSecretIdentifiersResponse = bitwardenClient.Secrets().List(organizationId);
```

# Delete secret or project

> ```

Reviewer: ```suggestion
```csharp
```

var responseForSecretsDeleteResponse = bitwardenClient.Secrets().Delete(new Guid[]{secretId});
var responseForProjectsDeleteResponse = bitwardenClient.Projects().Delete(new Guid[]{projectId});
```

> 
> [access-tokens help page]: https://bitwarden.com/help/access-tokens/

Reviewer: ```suggestion

```

Author: changed
